SPECS/binutils/binutils.spec

```diff
@@ -30,10 +63,61 @@ Requires:       %{name} = %{version}
 It contains the libraries and header files to create applications
 for handling compiled objects.
 
+%if %{build_cross}
+%package -n cross-%{name}-common
+Summary: Cross-compilation binutils documentation
+BuildArch: noarch
+
+%description -n cross-%{name}-common
+Documentation for the cross-compilation binutils package.
+%endif
+
+%do_package aarch64-linux-gnu %{build_aarch64}
+
 %prep
 %autosetup -p1
 
+function prep_target () {
+    local target=$1
+    local condition=$2
+
+    if [ $condition != 0 ]
+    then
+        echo $1 >> cross.list
+    fi
+}
+
+touch cross.list
```

Comment:
Just wondering if this file, cross.list, is deleted after build. We don't want it packaged into the rpm right?